is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
This seems a little complicated - consider removing the `slice` method entirely and just compare to the current element.